### ISSUE TITLE
ARM64 build for mac m1

### ISSUE THREAD

**Issue by kocas** [epam/TimeBaseClientPython#5]
Hi all, would it be possible to add arm64 build?

**Comment by alex-karpovich** [comment]
Greetings, gihub doesn't have appropriate runners.

**Comment by kocas** [comment]
> 

Makes sense. Would you be willing to help me build it myself? Maybe we could come up with a readme file for those who are interested. I think I need to be able to build the TimeBaseClientCpp repo first, but not sure if that is the way you'd recommend.
